calebwagner/python-server
# ch 13: updating a database row

#### Changes Made
1. Modified file `aniamls/request.py` to include updated `update_animal` functionality.
1. Modified file `request_handler.py` to include updated `do_PUT` functionality.

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-ch13
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. View code file.
   > Confirm file modifications are present as indicated above.
